8Knot/pages/contributions/visualizations/issue_assignment.py

```diff
@@ -124,6 +130,13 @@ def cntrib_issue_assignment_graph(repolist, interval):
         logging.warning(f"{VIZ_ID} - NO DATA AVAILABLE")
         return nodata_graph
 
+    # remove assignment data if assigned to a bot
+    if bot_switch:
+        df["bot"] = df["assignee"].isin(app.bots_list)
```

Comment:
Can we just drop the row rather than keeping the row and dropping the values?

Reply:
So in this case we do not want to drop the row only remove the assignee data. This visualization counts the number of issues with and without assignments so to drop the row would drop the existence of an issue if there is no other assignees besides the bot

Reply:
ah ++